### ISSUE TITLE
BUG: LFWA Critical Error

### ISSUE THREAD

**Issue by csmcallister** [DataKind-DC/capital-nature-ingest#222]
## Expected Behavior
Scraper works and, if it doesn't errors for individual events are caught and logged in `lfwa.py`.

## Current Behavior
traceback:

```
"02-25-2020","CRITICAL","Little Falls Watershed Alliance","Exception getting events in lfwa: 'Response' object has no attribute 'status'","<class 'AttributeError'> 'Response' object has no attribute 'status' File ""get_events.py"" line 39 in get_source_events    events = event_source_main()   File ""/Users/mcallistercs/Desktop/dev/capital-nature-ingest/events/lfwa.py"" line 175 in main    events = parse_event_divs(event_divs)   File ""/Users/mcallistercs/Desktop/dev/capital-nature-ingest/events/lfwa.py"" line 140 in parse_event_divs    soup_level_two = get_url(ORG_URL + event_website)   File ""/Users/mcallistercs/Desktop/dev/capital-nature-ingest/events/lfwa.py"" line 30 in get_url    f""Non-200 status code of {r.status} makign GET request to {url}"")"
```

## Possible Solution
Use `r.status_code` instead of `r.status`.

## Context
Easy fix for a big :bug:

**Comment by Frankie-Figz** [comment]
I think this should be fixed with the commits of the branch "lfwa-fix"

**Comment by csmcallister** [comment]
Fixed by cf21912
